dashboard/src/pages/servers/[id]/player.tsx

```diff
@@ -507,6 +531,7 @@ const Player: NextPageWithLayout = () => {
 
     return (
         <div className="player-page-container">
+            <audio id="audio" src="https://github.com/anars/blank-audio/raw/master/10-minutes-of-silence.mp3" style={{ display: 'none' }}></audio>
```

Comment:
hmm this is a hack, i wonder if there's anyway to do this without audio element.

Also it only works with chrome it seems, firefox won't set the media applet for me.

Reply:
I have checker playing just audio with JS, but it doesn't work. And regarding Firefox, that's weird newer version of Firefox should be compatible https://developer.mozilla.org/en-US/docs/Web/API/MediaSession#browser_compatibility